imap_processing/tests/mag/test_mag_decom.py

```diff
@@ -5,13 +5,16 @@
 
 from imap_processing.cdf import global_attrs
 from imap_processing.cdf.utils import write_cdf
-from imap_processing.mag.l0.decom_mag import decom_packets, export_to_xarray
+from imap_processing.mag import mag_cdf_attrs
+from imap_processing.mag.l0.decom_mag import decom_packets, generate_dataset
 
 
 def test_mag_decom():
     current_directory = Path(__file__).parent
     burst_test_file = current_directory / "mag_l0_test_data.pkts"
-    l0 = decom_packets(burst_test_file)
+    packets = decom_packets(str(burst_test_file))
```

Comment:
Does your decom_packets function require a string now, or could you keep that as Path?

Reply:
Good catch, copy+paste fail!